google/cloud/commerce/quickstart/README.md

```diff
@@ -0,0 +1,154 @@
+
+1. Run the example, changing the placeholder(s) to appropriate values:
+
+   ```bash
+   bazel run :quickstart -- [billing-account]
+   ```
+
+## Using with CMake
+
+> :warning: If you are using Windows or macOS there are additional instructions
+> at the end of this document.
+
+1. Install CMake. The package managers for most Linux distributions include a
+   package for CMake. Likewise, you can install CMake on Windows using a package
+   manager such as [chocolatey][choco-cmake-link], and on macOS using
+   [homebrew][homebrew-cmake-link]. You can also obtain the software directly
+   from the [cmake.org](https://cmake.org/download/).
+
+1. Install the dependencies with your favorite tools. As an example, if you use
+   [vcpkg](https://github.com/Microsoft/vcpkg.git):
+
+   ```bash
+   cd $HOME/vcpkg
+   ./vcpkg install google-cloud-cpp[core,commerce]
+   ```
+
+   Note that, as it is often the case with C++ libraries, compiling these
+   dependencies may take several minutes.
+
+1. Configure CMake, if necessary, configure the directory where you installed
+   the dependencies:
+
+   ```bash
+   cd $HOME/google-cloud-cpp/google/cloud/commerce/quickstart
+   cmake -S . -B .build -DCMAKE_TOOLCHAIN_FILE=$HOME/vcpkg/scripts/buildsystems/vcpkg.cmake
+   cmake --build .build
+   ```
+
+1. Run the example, changing the placeholder(s) to appropriate values:
+
+   ```bash
+   .build/quickstart [billing-account]
+   ```
+
+## Platform Specific Notes
+
+### macOS
+
+gRPC [requires][grpc-roots-pem-bug] an environment variable to configure the
+trust store for SSL certificates, you can download and configure this using:
+
+```bash
+curl -Lo roots.pem https://pki.google.com/roots.pem
+export GRPC_DEFAULT_SSL_ROOTS_FILE_PATH="$PWD/roots.pem"
+```
+
+### Windows
+
+Bazel tends to create very long file names and paths. You may need to use a
+short directory to store the build output, such as `c:\b`, and instruct Bazel
+to use it via:
+
+```shell
+bazel --output_user_root=c:\b build ...
+```
+
+gRPC [requires][grpc-roots-pem-bug] an environment variable to configure the
+trust store for SSL certificates, you can download and configure this using:
+
+```console
+@powershell -NoProfile -ExecutionPolicy unrestricted -Command ^
+    (new-object System.Net.WebClient).Downloadfile( ^
+        'https://pki.google.com/roots.pem', 'roots.pem')
+set GRPC_DEFAULT_SSL_ROOTS_FILE_PATH=%cd%\roots.pem
+```
+
+[authentication-quickstart]: https://cloud.google.com/docs/authentication/getting-started "Authentication Getting Started"
+[bazel-install]: https://docs.bazel.build/versions/main/install.html
+[choco-cmake-link]: https://chocolatey.org/packages/cmake
+[grpc-roots-pem-bug]: https://github.com/grpc/grpc/issues/16571
+[homebrew-cmake-link]: https://formulae.brew.sh/formula/cmake
+[howto-setup-dev-workstation]: /doc/contributor/howto-guide-setup-development-workstation.md
+[quickstart-link]: https://cloud.google.com/commerce/docs/quickstart
```

Comment:
You need to update this link.  Again, https://cloud.google.com/marketplace/docs/commitment-api-purchasing is the best I could find.